components/prize_pool/wikis/ageofempires/prize_pool_custom.lua

```diff
@@ -0,0 +1,81 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Opponent = require('Module:Opponent')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {10, 6, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.opponentLibrary = 'Opponent/Custom'
+	args.loadFlags = true
+	args.loadTeams = true
+
+	local prizePool = PrizePool(args)
+		:create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart,
+		Variables.varDefault('tournament_type')
+	)
+	if opponent.opponentData.type == Opponent.solo then
+		lpdbData.participantflag = opponent.opponentData.players[1].flag
+
+		-- legacy extradata, to be removed once unused
+		lpdbData.extradata.participantname = opponent.opponentData.players[1].displayName
+		lpdbData.extradata.participantteam = opponent.opponentData.players[1].team
+
+		if opponent.additionalData.LASTVS then
+			lpdbData.extradata.lastvsflag = opponent.additionalData.LASTVS.players[1].flag
+			lpdbData.extradata.lastvsname = opponent.additionalData.LASTVS.players[1].displayName
+		end
+
+		lpdbData.extradata.patch = Variables.varDefault('tournament_patch')
+		lpdbData.extradata.monthandday = placement.date:sub(-5)
```

Comment:
Any particular reason these two are set only Solos only?
`monthandday` shouldn't be used anymore. Use `[[date_month::' .. month .. ']] AND [[date_day::' .. day .. ']]` on the query side.

Reply:
No reason, good catch.
I'll try to find all usage of `monthandday` on the wiki, so this does not need to be set.

Reply:
There seems to be no usage of `monthandday` anyway, so removing it now.